### PR TITLE
Set-EntraAdministrativeUnit

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

File: module/Entra/AdditionalFunctions/Set-EntraAdministrativeUnit.ps1 [hunk 1]
```diff
@@ -0,0 +1,94 @@
+function Set-EntraAdministrativeUnit {
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Set-EntraAdministrativeUnit.ps1:1]: Add license information at the top of this file

File: test/module/Entra/Set-EntraAdministrativeUnit.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,33 @@
+BeforeAll {
```
Comment (KenitoInc) [inline test/module/Entra/Set-EntraAdministrativeUnit.Tests.ps1:1]: Add license information at the top of this file

File: module/Entra/AdditionalFunctions/Set-EntraAdministrativeUnit.ps1 (hunk 1 above)
Comment (v-pughosh) [reply module/Entra/AdditionalFunctions/Set-EntraAdministrativeUnit.ps1:4]: done

File: test/module/Entra/Set-EntraAdministrativeUnit.Tests.ps1 (hunk 2 above)
Comment (v-pughosh) [reply test/module/Entra/Set-EntraAdministrativeUnit.Tests.ps1:4]: done

**Review by KenitoInc — DISMISSED** [review]

**Review by KenitoInc — APPROVED** [review]

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 3]
```diff
@@ -26,34 +25,51 @@ Updates an administrative unit.
...
+ -ObjectId <String> 
  [-Description <String>] 
+ [-DisplayName <String>] 
  [<CommonParameters>]
 ```
 
 ## Description
-The Set-EntraAdministrativeUnit cmdlet updates an administrative unit in Microsoft Entra ID.
+
+The `Set-EntraAdministrativeUnit` cmdlet updates an administrative unit in Microsoft Entra ID. Specify `ObjectId` parameter for updates an administrative unit.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:36]:
```suggestion
The Set-EntraAdministrativeUnit cmdlet updates an administrative unit in Microsoft Entra ID. Use the `ObjectId` parameter to specify the unit to update.

In delegated scenarios, the signed-in user needs a supported Microsoft Entra role or a custom role with `microsoft.directory/administrativeUnits/allProperties/allTasks permission`. The least privileged role for this operation is Privileged Role Administrator
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 4]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
...
+
+- `-Description` - Specifies a description, which you want to update.
+
+### Example 2: Update DisplayName
 
 ```powershell
 Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
-Set-EntraAdministrativeUnit -Id 'bbbbbbbb-1111-2222-3333-cccccccccccc' -Description 'Updated Description'
+$params = @{
+    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:64]:
```suggestion
    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 5]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
 
 ```powershell
 Set-EntraAdministrativeUnit 
- -Id <String>
- [-DisplayName <String>] 
+ -ObjectId <String>
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:28]:
```suggestion
 -Id <String> 
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 6]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
...
 ## Examples
 
-### Example 1: Update the display name
+### Example 1: Update Description
 
 ```powershell
 Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
-Set-EntraAdministrativeUnit -Id 'bbbbbbbb-1111-2222-3333-cccccccccccc' -DisplayName 'New Updated Display Name'
+$params = @{
+    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:47]:
```suggestion
    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 7]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
...
+    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
+    DisplayName = 'UpdatedAU'
+}
+Set-EntraAdministrativeUnit @params
 ```
 
-This command updates the description of the specified administrative unit.
+This Command update DisplayName specific administrative unit.
+
+- `-ObjectId` - Specifies the ID of an administrative unit, which you want to update. In this example, `aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb` represents the ID of the administrative unit.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:72]:
```suggestion
- `-Id` - Specifies the ID of an administrative unit, which you want to update. In this example, `aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb` represents the ID of the administrative unit.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 8]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
...
+ -Id <String> 
  [-Description <String>] 
+ [-DisplayName <String>] 
  [<CommonParameters>]
 ```
 
 ## Description
-The Set-EntraAdministrativeUnit cmdlet updates an administrative unit in Microsoft Entra ID.
+
+The `Set-EntraAdministrativeUnit` cmdlet updates an administrative unit in Microsoft Entra ID. Use the `ObjectId` parameter to specify the unit to update.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:36]:
```suggestion
The `Set-EntraAdministrativeUnit` cmdlet updates an administrative unit in Microsoft Entra ID. Use the `Id` parameter to specify the unit to update.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 9]
```diff
@@ -26,34 +25,53 @@ Updates an administrative unit.
...
+    Description = 'Updated AU Description'
+}
+Set-EntraAdministrativeUnit @params
 ```
 
-This command updates the display name of the specified administrative unit.
+This Command update Description of specific administrative unit.
 
-### Example 2: Update the description
+- `-ObjectId` - Specifies the ID of an administrative unit, which you want to update. In this example, `aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb` represents the ID of the administrative unit.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:55]:
```suggestion
- `-Id` - Specifies the ID of an administrative unit, which you want to update. In this example, `aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb` represents the ID of the administrative unit.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md [hunk 10]
```diff
@@ -89,9 +107,9 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Id
+### -ObjectId
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraAdministrativeUnit.md:110]:
```suggestion
### -Id
```

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**